src/graphics/gpu/text.rs

```diff
@@ -43,6 +46,13 @@ impl TextRenderer {
         let cache_view =
             ArcTextureView::new(cache.create_view(&wgpu::TextureViewDescriptor::default()));
 
+        let cache_bind = BindGroupBuilder::new().image(&cache_view, wgpu::ShaderStages::FRAGMENT);
+        let cache_bind = ArcBindGroup::new(device.create_bind_group(&wgpu::BindGroupDescriptor {
+            label: None,
+            layout: &cache_bind_layout,
+            entries: cache_bind.entries(),
+        }));
+
```

Comment:
Is this done so that `TextRenderer` ownes it's cache image bind group? Similar to `Image`?

Reply:
Yes, since `InternalCanvas` no longer relies on the bind group cache to bind images, all images need to have their own bind group stapled to them. This would've also been leaking anyway as the text renderer's image is recreated whenever the glyph cache needs to be resized.